test/commands/schema/apply.test.js

```diff
@@ -2,134 +2,165 @@ const testCli = require('./../test-cli');
+        it('should exit with exit code 5', () => testCli({
           file: {
             chdir: '/tmp',
             name: './.forestadmin-schema.json',
-            content: forestadminSchemaSnake,
+            content: forestadminSchema,
           },
           env: testEnv2,
-          token: 'any',
-          api: [postSchema(postSchemaMatch)],
+          api: [postSchema503()],
           command: () => ApplySchemaCommand.run([]),
-          std: [
-            { out: 'Reading "./.forestadmin-schema.json"...' },
-            {
-              out: 'Using the forest environment secret found in the environment variable "FOREST_ENV_SECRET"',
-            },
-            { out: 'Sending "./.forestadmin-schema.json"...' },
-            { out: 'The schema is the same as before, nothing changed.' },
-          ],
+          std: [{ err: 'Forest is in maintenance for a few minutes. We are upgrading your experience in the forest. We just need a few more minutes to get it right.' }],
+          exitCode: 5,
+          token: 'any',
         }));
       });
+
+      describe('with forest server returning 200', () => {
+        describe('with a schema with camelcased keys', () => {
+          it('should send the schema', () => testCli({
+            file: {
+              chdir: '/tmp',
+              name: './.forestadmin-schema.json',
+              content: forestadminSchema,
+            },
+            env: testEnv2,
+            token: 'any',
+            api: [postSchema(postSchemaMatch)],
+            command: () => ApplySchemaCommand.run([]),
+            std: [
+              { out: 'Reading "./.forestadmin-schema.json"...' },
+              {
+                out: 'Using the forest environment secret found in the environment variable "FOREST_ENV_SECRET"',
+              },
+              { out: 'Sending "./.forestadmin-schema.json"...' },
+              { out: 'The schema is the same as before, nothing changed.' },
+            ],
+          }));
+        });
+
+        describe('with a schema with snakecased keys', () => {
+          it('should send the schema', () => testCli({
+            file: {
+              chdir: '/tmp',
+              name: './.forestadmin-schema.json',
+              content: forestadminSchemaSnake,
+            },
+            env: testEnv2,
+            token: 'any',
+            api: [postSchema(postSchemaMatch)],
+            command: () => ApplySchemaCommand.run([]),
+            std: [
+              { out: 'Reading "./.forestadmin-schema.json"...' },
+              {
+                out: 'Using the forest environment secret found in the environment variable "FOREST_ENV_SECRET"',
+              },
+              { out: 'Sending "./.forestadmin-schema.json"...' },
+              { out: 'The schema is the same as before, nothing changed.' },
+            ],
+          }));
+        });
+      });
     });
   });
+
+  describe('when the user is not logged in', () => {
```

Comment:
I would move this test on top of the file (before "when the user is logged in"),
because it is the first scenario to test in my opinion, not one of the last.

Reply:
Fixed in #100